lib/bolt/result.rb

```diff
@@ -107,6 +107,10 @@ def message
       @value['_output']
     end
 
+    def message?
```

Comment:
I'm not really sure how I feel about this... since `Bolt::Result#message` can return either `nil` or a string (including just whitespace or empty strings), this simplifies checking if there's a 'useful' message in the result.